sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/Utility.java

```diff
@@ -552,4 +553,19 @@ public static JobState toJobState(State jobState) {
             throw new RuntimeException(String.format("job state, %s is not supported.", jobState));
         }
     }
+
+    // TODO: Partial complete is still not well functional, https://github.com/Azure/azure-sdk-for-java/issues/18897
+    public static Map<String, Integer> parseTarget(String targetReference) {
```

Comment:
Is this used anywhere currently?

Reply:
Not. It is not used. I can delete it now.